app/proto/events.proto

```diff
@@ -280,12 +280,15 @@ message ListMyEventsReq {
 
   uint32 page_size = 5;
   string page_token = 6;
+  optional uint32 page_number = 9;
 }
 
 message ListMyEventsRes {
   repeated Event events = 1;
 
   string next_page_token = 2;
+  optional uint32 page_number = 3;
```

Comment:
Do you need to return the page number if it's just what was in the input?

Reply:
Good point, probably not. Fixed.